internal/controller/backup/periodic.go

```diff
@@ -114,6 +114,14 @@ var errEmptyList = errors.New("no items available to enqueue")
 // enqueueSchedulesOrIncompleteBackups enqueues the [github.com/K0rdent/kcm/api/v1alpha1.ManagementBackup] objects which
 // either are schedules or are not yet completed.
 func (r *Runner) enqueueSchedulesOrIncompleteBackups(ctx context.Context) error {
+	management := &kcmv1alpha1.Management{}
+	if err := r.cl.Get(ctx, client.ObjectKey{Name: kcmv1alpha1.ManagementName}, management); err != nil {
+		return fmt.Errorf("failed to get Management: %w", err)
+	}
+	if !management.DeletionTimestamp.IsZero() {
+		return nil
+	}
+
```

Comment:
there is inconsistency: this is a periodic runner, so "reconciles" continues execution each minute, so either add the same check in the Reconciler or remove this one

Reply:
missed the `internal/controller/management_backup_controller.go`